tests/unit/services/labwhere/client.spec.js

```diff
@@ -1,130 +1,240 @@
-import { getLabwhereLocations, scanBarcodesInLabwhereLocation } from '@/services/labwhere/client.js'
+import {
+  getLabwhereLocations,
+  scanBarcodesInLabwhereLocation,
+  exhaustLibraryVolumeIfDestroyed,
+} from '@/services/labwhere/client.js'
 import LabwhereLocationsFactory from '@tests/factories/LabwhereLocationsFactory.js'
+import * as pacbioLibraryUtilities from '@/stores/utilities/pacbioLibraries.js'
+import * as pacbioLibraryService from '@/services/traction/PacbioLibrary.js'
+import { createPinia, setActivePinia } from '@support/testHelper.js'
+import { beforeEach, describe, it } from 'vitest'
+const mockFetch = vi.fn()
+const labwhereLocationsFactory = LabwhereLocationsFactory()
 
-describe('client', () => {
-  const mockFetch = vi.fn()
-  const labwhereLocationsFactory = LabwhereLocationsFactory()
+beforeEach(() => {
```

Comment:
This is what I am referring to above.

We are essentially mocking fetch globally because of what we are doing in the client.